ArmPkg/Drivers/SmmVariablePei/SmmVariablePei.c

```diff
@@ -0,0 +1,231 @@
+/** @file -- SmmVariablePei.c
+  Provides interface for reading Secure System Variables during PEI.
+
+  Copyright (c) 2006 - 2019, Intel Corporation. All rights reserved.<BR>
+  Copyright (c) Microsoft Corporation.<BR>

```

Comment:
I don't know if we care to remove the `<BR>`

Reply:
I don't think it matters since the copyright is intact in any case and that just serves a formatting purpose depending on how the file content is rendered. If anyone has a strong opinion, I'll update it. Otherwise, will leave it.

Reply:
Let's remove this module. :)

Reply:
Going to close this for now. Will update if anyone replies wanting the change.

Reply:
Removed the module.